frontend/src/components/App/Layout.tsx

```diff
@@ -48,7 +51,14 @@ export default function Layout({}: LayoutProps) {
   const classes = useStyle();
   const arePluginsLoaded = useTypedSelector(state => state.ui.pluginsLoaded);
 
+  const dispatch = useDispatch();
   const { t } = useTranslation('frequent');
+
+  useEffect(() => {
+    const settings = JSON.parse(localStorage.getItem('settings') || '{}');
+    dispatch(setUISettings(settings));
```

Comment:
The localstorage's values could be set as the initial state in redux, so we'd skip having this dispatch here.

Reply:
done